public/app/features/admin/UserOrgs.tsx

```diff
@@ -150,39 +158,63 @@ class UnThemedOrgRow extends PureComponent<OrgRowProps, OrgRowState> {
     this.setState({ isChangingRole: false });
   };
 
+  onBuiltinRoleChange = (newRole: OrgRole) => {
+    this.props.onOrgRoleChange(this.props.org.orgId, newRole);
+  };
+
   render() {
-    const { org, isExternalUser, theme } = this.props;
+    const { user, org, isExternalUser, theme } = this.props;
     const { currentRole, isChangingRole } = this.state;
     const styles = getOrgRowStyles(theme);
     const labelClass = cx('width-16', styles.label);
     const canChangeRole = contextSrv.hasPermission(AccessControlAction.OrgUsersRoleUpdate);
     const canRemoveFromOrg = contextSrv.hasPermission(AccessControlAction.OrgUsersRemove);
+    const rolePickerDisabled = isExternalUser || !canChangeRole;
 
     const inputId = `${org.name}-input`;
     return (
       <tr>
         <td className={labelClass}>
           <label htmlFor={inputId}>{org.name}</label>
         </td>
-        {isChangingRole ? (
+        {config.licenseInfo.hasLicense ? (
```

Comment:
Could we maybe have the new role picker for OSS as well, but just show the built in roles there?

Reply:
Yes, maybe. We also need to show other roles if access control enabled.

Reply:
We decided to leave picker as is in OSS.